services/graph/pkg/service/v0/tags.go

```diff
@@ -0,0 +1,226 @@
+package svc
+
+import (
+	"encoding/json"
+	"net/http"
+	"strings"
+
+	rpc "github.com/cs3org/go-cs3apis/cs3/rpc/v1beta1"
+	provider "github.com/cs3org/go-cs3apis/cs3/storage/provider/v1beta1"
+	revaCtx "github.com/cs3org/reva/v2/pkg/ctx"
+	revactx "github.com/cs3org/reva/v2/pkg/ctx"
+	"github.com/cs3org/reva/v2/pkg/events"
+	"github.com/cs3org/reva/v2/pkg/storagespace"
+	revaTags "github.com/cs3org/reva/v2/pkg/tags"
```

Comment:
Curious: Why alias this with `revaTags`? Whats wrong with `tags`?

Reply:
assignment.Tags is a slice of tags ([]string), tags package does not have a `fromSlice` func at the moment.